srcpkgs/maxima/template

```diff
@@ -1,18 +1,17 @@
-# Template file for 'maxima'.
+# Template file for 'maxima'
 pkgname=maxima
-version=5.43.2
+version=5.44.0
 revision=1
 build_style=gnu-configure
 configure_args="$(vopt_enable clisp) $(vopt_enable sbcl)"
-hostmakedepends="python perl emacs texinfo"
-makedepends="$(vopt_if clisp clisp) $(vopt_if sbcl sbcl)"
+hostmakedepends="python perl emacs texinfo $(vopt_if clisp clisp) $(vopt_if sbcl sbcl)"
```

Comment:
I think it give wrong output from lisp. If you do native build of sbcl, you can likely do native build of maxima too.